main.go

```diff
@@ -138,6 +139,28 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
+func createStorageHandle(flags *flagStorage) (storageHandle storage.StorageHandle, err error) {
+	var tokenSrc oauth2.TokenSource
+
+	tokenSrc, err = auth.GetTokenSource(context.Background(), flags.KeyFile, flags.TokenUrl, true)
```

Comment:
here also can you do tokenSrc, err := instead of declaring tokenSrc as a variable on top.

Reply:
fixed